Cargo.toml

```diff
@@ -31,3 +39,13 @@ features = [
     "Win32_UI_WindowsAndMessaging",
     "Win32_System_SystemServices",
 ]
+
+[[example]]
+name = "list_displays_async"
+path = "examples/list_displays_async.rs"
+required-features = ["async"]
+
+[[example]]
+name = "set_brightness_async"
+path = "examples/set_brightness_async.rs"
+required-features = ["async"]
```

Comment:
These examples must be specified here with the required features, otherwise they will fail to build when running `--no-default-features`